protocol/daemons/pricefeed/client/types/price_feed_mutable_market_configs.go

```diff
@@ -264,8 +277,9 @@ func (pfmmc *PricefeedMutableMarketConfigsImpl) ValidateAndTransformParams(marke
 			if exchangeConfig.AdjustByMarket != "" {
 				adjustByMarketId, ok := marketNameToId[exchangeConfig.AdjustByMarket]
 				if !ok {
-					return nil, nil, fmt.Errorf(
-						"invalid exchange config json for exchange '%v' on market param %v: adjustByMarket '%v' not found",
+					return nil, nil, nil, fmt.Errorf(
```

Comment:
Does this mean if gov passes a market with unsupported/mis-spelled exchange, it would still block other valid market updates?

Reply:
The errors in this loop should be unreachable because of the validation that happens before this loop is entered - see the comment on line 258, and also "unexpected internal error". In this case I am ok to return wholesale because a.) this is specifically already checked for, and should not happen, and b.) the `MarketParam` update could be partially but not fully applied if we continue, and that puts the daemon into a state that's difficult to predict. I prefer for each `MarketParam` update to succeed or fail atomically.

Reply:
Ah I see, I missed the validation on line 248.